core/src/main/java/org/apache/iceberg/MicroBatches.java

```diff
@@ -145,7 +158,7 @@ public MicroBatch generate(int startFileIndex, long targetSizeInBytes, boolean i
     /**
      * Method to skip the manifest file in which the index is smaller than startFileIndex. For example, if the
      * index list is : (m1, 0), (m2, 3), (m3, 5), and startFileIndex is 4, then the returned manifest index list is:
-     * (m2, 3), (m3, 5).
+     * (m3, 5).
```

Comment:
This update to this java doc comment is what inspired this PR.

Reply:
The change is not correct. If the `startFileIndex` is 4, then "(m2, 3)" should not be skipped, the 4th index is included in "(m2, 3)"